.golangci.yml

```diff
@@ -131,6 +131,9 @@ linters:
     - makezero
     - nolintlint
     - gocyclo
+    - nosnakecase
+    - interfacebloat
+    - revive
```

Comment:
This generates errors like this: we could also add package comments, but I don't know if it is really useful and I think that we are missing a lot of them.
```
util/observable/action/action.go:15:1: package-comments: should have a package comment (revive)
package action
```